lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -2098,7 +2097,7 @@
 
     // A helper function that is meant to rethrow SOE and OOM exceptions allowing them to propagate.
     var throwExIfOOMOrSOE = function (ex) {
-        if (ex.number === -2146828260 || ex.number === -2146828281) {
+        if (ex.number === Number("-2146828260") || ex.number === Number("-2146828281")) {
```

Comment:
these Number() calls should probably get comments explaining why theyre not just number literals.